### PR TITLE
Ant Smasher game

### LINKED ISSUE #4334: [New game]: Ant killing game
### 🎮 Game Request

Lots of ants will be coming from random side of the screen. user needs to press the ant so that it will kill the ant and score is stored in local storage.

### Point down the features

This game will be addictive because the ant speed increases when score increases.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by mukilan2815**
## PR Description 📜
Ant smasher game which users can play 

Fixes #4334 

<hr>
 
## Mark the task you have completed ✅

- [x ] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x ] I have commented my code, particularly in hard-to-understand areas.
- [x ] My changes generates no new warnings.
- [x ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x ] I have added screenshot for website preview in assets/images 
- [x ] I have added entries for my game in main README.md
- [x ] I have added README.md in my folder 
- [] I have added working video of the game in README.md (optional)
- [ x] I have specified the respective issue number for which I have requested the new game.

<hr>

image

--- 
<br>
GSSOC'24

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @mukilan2815 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please add your game image in a asset folder of the gamezone

**Comment by mukilan2815** [comment]
@ishita-43 Completed

**Comment by mukilan2815** [comment]
@ishita-43 please check

**Comment by mukilan2815** [comment]
@kunjgit @ishita-43

File: README.md
Comment (ishita-43) [inline README.md:1]: pLEASE MAKE SURE YOU ADDED YOUR GAME ENTRY IN THIS FILE

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md
Comment (mukilan2815) [reply README.md:1]: i have already added that

File: README.md [hunk 1]
```diff
@@ -361,17 +363,16 @@ This repository also provides one such platforms where contributers come over an
...
 | [Disney_Trivia](https://github.com/manmita/GameZone/tree/Disney_Trivia/Games/Disney_Trivia)|
 |[Harmony_Mixer](https://github.com/kunjgit/GameZone/tree/main/Games/Harmony_Mixer)|
 | [Helicopter_Game](https://github.com/kinjgit/GameZone/tree/main/Games/Helicopter_Game) |
 | [Bouncing Ball Game](https://github.com/kunjgit/GameZone/tree/main/Games/Bouncing_Ball_Game) |
 | [Disney_Trivia](https://github.com/manmita/GameZone/tree/Disney_Trivia/Games/Disney_Trivia)|
 |[Harmony_Mixer](https://github.com/kunjgit/GameZone/tree/main/Games/Harmony_Mixer)|
 |[Steam_Punk](https://github.com/kunjgit/GameZone/tree/main/Games/Steam_Punk)|
-|[Tower Defence Game](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Tower_Defence_Game)|
-
+|[Tower Defence Game](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Tower_Defence_Game) | | [Ant Smasher Game](https://github.com/mukilan2815/GameZone/tree/main/Games/Ant_Smasher)|
```
Comment (mukilan2815) [inline README.md:375]: added here check it out

Comment (ishita-43) [reply README.md:375]: you need to add kunjgit instead of you own repo link to make it work please update this

Comment (mukilan2815) [reply README.md]: its been too long

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by ishita-43** [comment]
Please resolve merge conflicts

**Comment by mukilan2815** [comment]
Done @ishita-43

**Comment by kunjgit** [comment]
Thank you  @mukilan2815 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**